benchmarks/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+
+const fs = require("fs");
+const Promise = require("bluebird");
```

Comment:
The struggle is real:
- I put in the wrong directory for find and replace 🤦‍♂️ 
- ESLint doesn't catch this because `Promise` _is_ a used variable 😐